charts/kube-otel-stack/values.yaml

```diff
@@ -57,10 +57,11 @@ tracesCollector:
   enabled: false
   name: traces
   clusterName: ""
-
   image: otel/opentelemetry-collector-contrib:0.88.0
   mode: deployment
   replicas: 1
+  labels:
+    opentelemetry.io/opamp-managed: "true"
```

Comment:
better to require users to specify this only on collectors they want to be managed, or include it for them automatically if the opamp bridge is enabled?

Reply:
thinking "out loud": because this is our recommended config, I think we get to be opinionated here. and we believe that customers _should_ have all collectors report config and status through the opamp bridge. so I'd advocate for configuring each collector to be opamp-managed by default.

I don't know why a customer want to have any individual collector not be managed by the opamp bridge. But I suppose the most robust solution gives the users an easy toggle to allow the collector to opt out

Reply:
this should be explicitly set on each collector a user wants to be opamp managed. instead of expecting users to fuss around with custom labels i added an attribute to collectors in `values.yaml` which we read to set the label in `collector.yaml`. we could make this true by default (only has an impact when the bridge is in use)

Reply:
oops my reply just now was also on an old unrefreshed page where i hadn't read your reply greg

Reply:
im gonna fall back to jacob's advice [here](https://github.com/lightstep/otel-collector-charts/pull/76#discussion_r1526416491) and set `opamp-reporting=true` and omit opamp-managed by default

context— 

`opamp-reporting:true` tells the bridge to only report on a given collector, and not mutate it

`opamp-managed:true` tells the bridge it can report on and mutate a given collector

these labels only do something if `true`. we omit the label rather than set labels to `false` with this chart, but someone manually setting a label to `false` would have no effect

both shouldn't be true, but if they are then `opamp-reporting` takes precedence and we won't modify that collector ([code link](https://github.com/open-telemetry/opentelemetry-operator/blob/658b9c54836ebe35e11c0bff8dd519d60108d4aa/cmd/operator-opamp-bridge/operator/client.go#L134))